docs/manuals/source/TasksAndConcepts/Plugins.rst

```diff
@@ -1812,11 +1826,128 @@ Troubleshooting
+
+Documentation of mariabackup is available online: https://mariadb.com/kb/en/mariabackup/.
+
+It is stable since MariaDB 10.1.48
+
+
+Prerequisites of mariabackup Plugin
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The :command:`mariabackup` binary needs to be installed on the |fd|. refer to previous documentation link.
+
+For authentication the :file:`.mycnf` file of the user running the |fd| is used. Before proceeding, make sure that mariabackup can connect to the database, create backups and is able to restore.
+
+
+Installation of mariabackup Plugin
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Make sure you have met the prerequisites, then install the package **bareos-filedaemon-mariabackup-python-plugin**.
+
+Configuration of mariabackup Plugin
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Activate your plugin directory in the |fd| configuration. See :ref:`fdPlugins` for more about plugins in general.
+
+.. code-block:: bareosconfig
+   :caption: bareos-fd.d/client/myself.conf
+
+   Client {
+     ...
+     Plugin Directory = /usr/lib64/bareos/plugins
+     Plugin Names = "python"
+   }
+
+Now include the plugin as command-plugin in the Fileset resource:
+
+.. code-block:: bareosconfig
+   :caption: bareos-dir.d/fileset/mariadb.conf
+
+   FileSet {
+       Name = "mariadb"
+       Include  {
+           Options {
+               signature = MD5
+           }
+           #...
+           Plugin = "python"
+                    ":module_path=/usr/lib64/bareos/plugins"
+                    ":module_name=bareos-fd-mariabackup"
+                    ":mycnf=/root/.my.cnf"
+       }
+   }
+
+The plugin will call :command:`mariabackup` to create a backup stream of all databases in the xbstream format. This stream will be processed by Bareos. Full backups can be made for all table formats, while incremental backups are only supported for **InnoDB** tables. Incremental backups for other table formats will create a full backup.
+
+You can append options to the plugin call as key=value pairs, separated by ’:’. The following options are available:
+
+-  With :strong:`mycnf` you can make mariabackup use a special mycnf-file with login credentials.
+
+-  :strong:`dumpbinary` lets you modify the default command mariabackup.
+
+-  :strong:`dumpoptions` to modify the options for mariabackup. Default setting is: :command:`--backup --stream=xbstream --extra-lsndir=/tmp/individual_tempdir`
+
+-  :strong:`restorecommand` to modify the command for restore. Default setting is: :command:`mbstream -x -C`
+
+-  :strong:`strictIncremental`: By default (false), an incremental backup will create data, even if the Log Sequence Number (LSN) was not increased since last backup. This is to ensure, that eventual changes to MYISAM/ARIA/Rocks tables get into the backup. MYISAM/ARIA/Rocks does not support incremental backups, you will always get a full backup of these tables. If set to true, no data will be written into backup, if the LSN was not changed.
```

Comment:
```suggestion
-  :strong:`strictIncremental`: By default (false), an incremental backup will create data even if the Log Sequence Number (LSN) was not increased since last backup. This is to ensure that eventual changes to MYISAM/ARIA/Rocks tables get into the backup. MYISAM/ARIA/Rocks does not support incremental backups, you will always get a full backup of these tables. If set to true, no data will be written into backup, if the LSN was not changed.
```